src/hooks/usePlanByEpg.ts

```diff
@@ -24,16 +24,63 @@ const usePlanByEpg = (channels: EpgChannel[]) => {
     ];
   }, [channels]);
 
+  // Theme configuration for the Planby EPG with only the colors set that are used
+  // Fixed values are used because of the tecnical dept we have with the current setup of the config colors values and the theme values
```

Comment:
```suggestion
  // Theme configuration for the Planby EPG with only the colors set that are used
  // Fixed values are used because the default highlightColor and backgroundColor are only available in SCSS
```